lib/csv/fields_converter.rb

```diff
@@ -44,7 +44,7 @@ def empty?
       @converters.empty?
     end
 
-    def convert(fields, headers, lineno)
+    def convert(fields, headers, lineno, quoted_fields = nil)
```

Comment:
Can we make `quoted_fields` mandatory argument? If we use `nil` here, `FieldInfo#quoted?` returns `nil` not boolean.

Reply:
Thanks. I thought it is not required when converting headers.
But it is better to explicitly pass an argument for them.

Reply:
Headers may be parsed from string. So quoted information for headers may be useful.

Reply:
I understood, thank you. So currently no information is passed in `#adjust_headers`  ....

Reply:
I added support for quoted header converters.